buildingmotif/database/table_connection.py

```diff
@@ -377,10 +377,21 @@ def update_db_template_optional_args(
         )
         db_template.optional_args = optional_args
 
-    def add_template_dependency(
+    def add_template_dependency_preliminary(
```

Comment:
Test?

Reply:
This is tested by the existing template dependency tests